### PR TITLE
Correct path for custom 404 and 500 error pages

### PR DESCRIPTION
**Pull request by jainarunarunjain**
@deshraj Please review this PR

### CODE REVIEW

File: settings/common.py [hunk 1]
```diff
@@ -83,7 +83,7 @@
 
 ROOT_URLCONF = 'evalai.urls'
 
-TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend')
+TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend/dist')
```
Comment (deshraj) [inline settings/common.py]: Can you please rewrite this as `TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend', 'dist')`

Comment (jainarunarunjain) [inline settings/common.py]: @deshraj sure

**Comment by jainarunarunjain** [comment]
@deshraj Can you please tell me why this build still fail.

**Comment by deshraj** [comment]
You have to change the path here https://github.com/Cloud-CV/EvalAI/blob/master/evalai/views.py#L5

**Comment by deshraj** [comment]
Also, you can run tests on your local machine using the command `py.test`. So, there is no need to push and then wait for this to complete.

**Pull request closed without merging**